phpunit.xml.dist

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit colors                 = "true"
+	 bootstrap              = "vendor/autoload.php"
```

Comment:
There is not .travis.yml file, so having tests without automating them is kinda pointless. You could add one with a build-matrix but you'll have to run all tests for that